connectors/apache-connector/src/main/java/org/glassfish/jersey/apache/connector/ApacheConnector.java

```diff
@@ -249,6 +251,34 @@
             }
         }
 
+        Object redirectStrategy = config.getProperties().get(ApacheClientProperties.REDIRECT_STRATEGY);
+        if (redirectStrategy != null) {
+            if (!(redirectStrategy instanceof RedirectStrategy)) {
+                LOGGER.log(
+                    Level.WARNING,
+                    LocalizationMessages.IGNORING_VALUE_OF_PROPERTY(
+                        ApacheClientProperties.REDIRECT_STRATEGY,
+                        redirectStrategy.getClass().getName(),
+                        RedirectStrategy.class.getName())
+                );
+                redirectStrategy = null;
+            }
+        }
+
+        Object defaultAuthSchemeRegistry = config.getProperties().get(ApacheClientProperties.DEFAULT_AUTH_SCHEME_REGISTRY);
```

Comment:
can this variable be named without ``default`` word? For reasons described in previous comment.